src/OrchardCore.Modules/OrchardCore.Media/Services/MediaCacheManager.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace OrchardCore.Media.Services
+{
+    public class MediaCacheManager : IMediaCacheManager
+    {
```

Comment:
Just a start towards some cache management.

Reply:
This could be implementation specific. Purge is fine as this should be a minimum implementation. But other UI elements could be on a per module basis, i.e. knowing how the storage works. For instance a store could be based on sqlite and have all the metadata necessary to provide a way to remove files based on FIFO, or MRU or size based.